routes/student.js

```diff
@@ -18,14 +18,9 @@ var storage = multer.diskStorage({
 
 var upload = multer({ storage: storage });
 
-router.get("/:sid", (req, res, next) => {
-	Student.findById(req.params.sid).exec((err,result) => {
-		if (err)
-		{
-			return next(err);
-		}
-		res.json(result);
-	})
```

Comment:
Add this code to branch

Reply:
Added the code to teacher-routes-2 branch